tests/test_issue_1_cli_bank_selection.py

```
@@ -0,0 +1,102 @@
+"""
+
+import os
+import sqlite3
+import subprocess
+import sys
+from pathlib import Path
+
+ROOT = Path(__file__).resolve().parent.parent
+
+
+def _run_cli(args, tmp_path, extra_env):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_run_cli`

(ANN202)